src/assertions/contain.js

```diff
@@ -1,8 +1,11 @@
+import reactElementToJSXString from 'react-element-to-jsx-string'
+
 export default function contain ({ wrapper, markup, arg1, sig }) {
+  const arg1JSXString = reactElementToJSXString(arg1)
   this.assert(
     wrapper.hasNode(arg1),
-    () => 'expected ' + sig + ' to contain #{exp} ' + markup(),
-    () => 'expected ' + sig + ' not to contain #{exp} ' + markup(),
+    () => 'expected ' + sig + ' to contain ' + arg1JSXString + markup(),
+    () => 'expected ' + sig + ' not to contain ' + arg1JSXString + markup(),
```

Comment:
I agree, I think it is too, however I think the markup would be even more useful if it showed the JSX of what's in the wrapper rather than final rendered markup. That wasn't in the scope of this PR though :P